tests/test_hydrology.py

```diff
@@ -151,3 +151,55 @@ def test_simple(self, q_series):
         out = xci.low_flow_frequency(q, 0.2, freq="YS")
 
         np.testing.assert_array_equal(out, [20, 0])
+
+
+class TestAntecedentPrecipitationIndex:
+    def test_simple(self, pr_series):
+        a = np.ones(50) * 10
+        a[15:20] = 20
+        a[35:40] = 0
+        pr = pr_series(a)
+        window = 7
+        out = xci.antecedent_precipitation_index(pr, window=window, p_exp=0.935)
```

Comment:
```suggestion
        out = xci.antecedent_precipitation_index(pr)
```
I suggest that at least one test uses the "defaults" so we test that these haven't changed.